test/helpers.go

```diff
@@ -88,19 +88,18 @@ func RandomReader(seed, size int) *bytes.Reader {
 
 // SetupTarTestFixture extracts the tarFile to outputDir.
 func SetupTarTestFixture(t testing.TB, outputDir, tarFile string) {
-	err := System("sh", "-c", `(cd "$1" && tar xzf - ) < "$2"`,
-		"sh", outputDir, tarFile)
+	f, err := os.Open(tarFile)
+	defer f.Close()
 	OK(t, err)
-}
 
-// System runs the command and returns the exit code. Output is passed on to
-// stdout/stderr.
-func System(command string, args ...string) error {
-	cmd := exec.Command(command, args...)
+	cmd := exec.Command("tar", "xzf", "-")
```

Comment:
No problem. The only issue I am seeing is that os.RemoveAll(tempdir) fails because the tar file contains readonly files, which Windows cannot delete due to https://github.com/golang/go/issues/9606